### PR TITLE
Bump version to 17.13.0

### PR DESCRIPTION
**Pull request by DilwoarH**
Changes:
- Add support for description text (PR #971)
- Stop duplicate links within feedback form (PR #981)
- Add spacing to the feedback form (PR #981)

### CODE REVIEW

**Comment by alex-ju** [comment]
Can we get [this fix](https://github.com/alphagov/govuk_publishing_components/pull/981) in too after is merged?

**Comment by vanitabarrett** [comment]
@DilwoarH @alex-ju I'm just adding a fix to that PR, please don't merge yet 🙂 Should be ready soon.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+
+## 17.13.0
+
+* Add support for description text
```
Comment (leenagupte) [inline CHANGELOG.md]: The other entries have a like to the PR. Is that the standard for this repo?

Comment (DilwoarH) [reply CHANGELOG.md]: Yes sorry leena - added

**Pull request merged**